app/build.gradle

```diff
@@ -68,7 +68,7 @@ dependencies {
     exclude group: 'com.google.android', module: 'android'
   }
   // eclair core
-  compile 'fr.acinq.eclair:eclair-core_2.11:0.2-android-beta2'
+  compile 'fr.acinq.eclair:eclair-core_2.11:0.2-android-beta3-SNAPSHOT'
```

Comment:
The wallet should only use non SNAPSHOT dependencies to make sure tests are consistent.

Reply:
Let's wait for the 0.2-android-beta3 final version